scripts/stop_server

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
I'm assuming you'll fill in code to kill the `npm serve` later right?

Reply:
Nope, but might use it to perform some other instance cleanup, if required.
The start_scripts uses  "npm run serve" which uses "pm2 reload" so it's compatible for new and redeployments on the same instance.

Reply:
ok my bad! it's better to use start and stop for pm2 separately, will do this change.. thanks

Reply:
thanks for pointing it out.. verified in dev env and prod env.. with 1st time deployment and redeployment.
![image](https://user-images.githubusercontent.com/17198963/116754488-ab225880-a9bd-11eb-88f3-f94d0a33d04a.png)